src/assets.rs

```diff
@@ -111,8 +111,17 @@ impl HighlightingAssets {
             );
         }
 
+        let syntax_set = syntax_set_builder.build();
+        let missing_contexts = syntax_set.find_unlinked_contexts();
+        if missing_contexts.len() > 0 {
```

Comment:
Nice! Just fix this lint and this PR is golden:

```
warning: length comparison to zero
   --> src/assets.rs:116:12
    |
116 |         if missing_contexts.len() > 0 {
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^ help: using `!is_empty` is clearer and more explicit: `!missing_contexts.is_empty()`
    |
    = note: `#[warn(clippy::len_zero)]` on by default
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#len_zero
```

I did a test-run of the code and it seemed to work fine for me too :)